libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -578,17 +566,53 @@ - (void) webView:(WKWebView *) webView decidePolicyForNavigationAction:(WKNaviga
                  }
              }];
             shouldAllowRequest = NO;
-        } else if (self.navWebViewDelegate) {
-            [self.navWebViewDelegate webView:webView decidePolicyForNavigationAction:navigationAction decisionHandler:decisionHandler];
+        } else {
+            [self defaultWKNavigationHandling:webView decidePolicyForNavigationAction:navigationAction decisionHandler:decisionHandler];
             return;
         }
     }
-    
-    if(shouldAllowRequest)
+    if (shouldAllowRequest) {
         decisionHandler(WKNavigationActionPolicyAllow);
-    else
+    } else {
         decisionHandler(WKNavigationActionPolicyCancel);
-    
+    }
+}
+
+- (void) defaultWKNavigationHandling:(WKWebView *) webView decidePolicyForNavigationAction:(WKNavigationAction *) navigationAction decisionHandler:(void (^)(WKNavigationActionPolicy)) decisionHandler {
```

Comment:
The plugin requires us to do some redirect handling that's different from their `UIWebView` plugin due to structural differences so that handling is done here. This is the only part of my old plugin that survived.